src/app/containers/Include/index.jsx

```diff
@@ -18,13 +20,35 @@ const IncludeContainer = ({ html, type }) => {
     return null;
   }
 
+  const requireIncludeTypes = ['vj', 'idt1'];
```

Comment:
I feel this line and the following section would really benefit from some kind of explanatory comment, either inline or as a README.

Reply:
Should be addressed here: 65b10d92e21d85bb90682ff89cd094882371d5c9